packages/autorest.typescript/test/integration/generated/additionalProperties/package.json

```diff
@@ -29,7 +29,7 @@
     "rimraf": "^3.0.0",
     "dotenv": "^16.0.0",
     "@azure/identity": "^2.0.1",
-    "@azure-tools/test-recorder": "^2.0.0",
+    "@azure-tools/test-recorder": "^3.0.0",
```

Comment:
@HarshaNalluru Could you help release the 3.0.0 version so that our pipeline could work?